UPGRADE-7.1.md

```diff
@@ -68,6 +68,24 @@ All service classes that are loaded by `ServiceLocator::get()` must now be `publ
 # Cleanup
```

Comment:
Cronjobs node should be here after Make Legacy Services Public.

And probably be called something like: Correct Cronjob Hierarchy

Reply:
alright, position changed.